tests/common/plugins/sanity_check/checks.py

```diff
@@ -584,7 +584,7 @@ def _check_monit_on_dut(*args, **kwargs):
 @pytest.fixture(scope="module")
 def check_processes(duthosts):
     def _check(*args, **kwargs):
-        result = parallel_run(_check_processes_on_dut, args, kwargs, duthosts, timeout=600)
+        result = parallel_run(_check_processes_on_dut, args, kwargs, duthosts, timeout=1000)
```

Comment:
Can we add a check for `is multi-asic` AND `is KVM` before increasing the timeout? If the issue is not seen on physical devices then I think 600s is acceptable timeout, and a failure may be actually meaningful.

If agreed, please also add code-comments for why there are two diff timeouts for kvm and physical setups.

Reply:
@vaibhavhd if there is a timeout seen on any other set up, will that be meaningful?
Right now, it looks like for multi-asic vs, the system could be overloaded causing the delay.
But on a regular DUT or single asic VS, we do have tests separately to check cpu/memory load. So do we really need this timeout failure to understand any other specific failure.

Reply:
I am not sure, tbh. As I understand this increased timeout is added for below two reasons:
1) On multi-asic VS testbed, due to the increased number of services running, we see sanity check not able to complete critical process check before the 600s timeout.
2) KVM is slow.

The above two conditions do not hold true for non-multi-asic physical testbeds. So I think if there is a failure seen, there might be a different cause than the reasons listed above. In which case, we should analyze the cause and do not let increased timeout mask the failure.

Reply:
modified as per comment. Modify timeout only for multi-asic kvm.